Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -24,6 +24,11 @@
 #include "BaseTarget.xcconfig"
 
 DEFINES_MODULE = YES;
+ENABLE_MODULE_VERIFIER = $(USE_INTERNAL_SDK);
+OTHER_MODULE_VERIFIER_FLAGS = $(OTHER_MODULE_VERIFIER_FLAGS_$(WK_EMPTY_$(WK_OVERRIDE_FRAMEWORKS_DIR)));
+OTHER_MODULE_VERIFIER_FLAGS_ = -- -iframework $(WK_PRIVATE_SDK_DIR)$(WK_OVERRIDE_FRAMEWORKS_DIR) -isystem $(WK_PRIVATE_SDK_DIR)$(WK_ALTERNATE_WEBKIT_SDK_PATH)$(WK_LIBRARY_HEADERS_FOLDER_PATH);
+MODULE_VERIFIER_VERBOSE = NO;
```

Comment:
NO is the default, don't need to specify this

Reply:
I would actually leave it specified since it makes it easier to find/enable when debugging in the future.

Reply:
Since it doesn't cost much to have it, I'll leave this here to make it clear to other folks trying to debug failures in this area in the future (but are not familiar with this as I) that this helpful option is available.